addons/stock/views/stock_scrap_views.xml

```diff
@@ -171,6 +171,7 @@
                                 <field name="product_uom_category_id" invisible="1"/>
                                 <field name="product_uom_id" attrs="{'readonly': [('tracking', '=', 'serial')]}" groups="uom.group_uom"/>
                             </div>
+                            <field name="should_replenish"/>
```

Comment:
What's the point of this change? :thinking:

Reply:
to move fields Replenish quantities on left side.